docs/azure/vs-azure-tools-resources-managing-with-cloud-explorer.md

```diff
@@ -1,22 +1,24 @@
 ---
 title: Managing Azure resources with Cloud Explorer | Microsoft Docs
 description: Learn how to use Cloud Explorer to browse and manage Azure resources within Visual Studio.
-author: ghogen
+author: ghogen, robertmclaws
 manager: jmartens
 ms.technology: vs-azure
 ms.workload: azure-vs
 ms.topic: conceptual
-ms.date: 03/25/2017
-ms.author: ghogen
+ms.date: 01/22/2022
+ms.author: ghogen, robertmclaws
 ---
 
 # Manage the resources associated with your Azure accounts in Visual Studio Cloud Explorer
 
 ::: moniker range=">=vs-2022"
 > [!Important]
 > Cloud Explorer has been retired in Visual Studio 2022. Instead, you can use the following alternatives:
+> - To debug an Azure App Services or Azure Function apps, you can use the Connected Services or the Publish features available on all Projects. 
+>   -  If you rely on Visual Studio for publishing, you may access this feature under the "Additional Options" in the Hosting section. 
+>   -  If you don’t use the publishing feature, the [Connected Services](https://docs.microsoft.com/en-us/visualstudio/azure/overview-connected-services?view=vs-2022) node inside each Project in your Solution can help you configure access to Azure resources and invoke  various operations like remote debugging, remote profiling, starting/stopping the application, view streaming logs, and more.
```

Comment:
```suggestion
>   -  If you don’t use the publishing feature, the [Connected Services](overview-connected-services.md) node inside each Project in your Solution can help you configure access to Azure resources and invoke  various operations like remote debugging, remote profiling, starting/stopping the application, view streaming logs, and more.
```

Reply:
That the internal link format which would be appropriate for a local file.

Reply:
Will it use the correct View in that situation? Want to make sure it defaults to the right version of VS in that situation.